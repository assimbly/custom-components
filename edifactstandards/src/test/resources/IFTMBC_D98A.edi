UNA:+.? 'UNB+UNOA:1+EGDHAM+ICT+180704:1310+20180704327110'UNH+560800426466+IFTMBC:D:98A:UN:EGDHBG'BGM+770+560800426466+9'DTM+137:201807041310:203'TSR+30'LOC+73+DEHAM::6'LOC+91+DEHAM::6'RFF+BN:560800426466'DTM+179:20180703:102'RFF+CT:SQK500212'RFF+RF:807/USFA/00253'RFF+FF:169124'TCC+::::P'TDT+20+0986-021E++++++TMAN:146::THALASSA MANA'DTM+133:20180712:102'LOC+10+DEHAM::6'LOC+13+LKCMB::6'LOC+9+DEHAM::6'LOC+12+INMUN::6'LOC+7+INMUN::6'NAD+CZ+DEI000109+ICT INTERNATIONALE CONTAINER TRANSP:ORT GMBH:SPERBERWEG 6 A+++++41468+DE'GID+1+1:22G0::87'TMD+3++2'LOC+98+DEHBGH::6'FTX+AAA+++3 COLLI, POLYPROPYLENE'MEA+AAE+G+KGM:3370'MEA+AAE+AAW+MTQ:0'UNT+27+560800426466'UNZ+1+20180704327110'