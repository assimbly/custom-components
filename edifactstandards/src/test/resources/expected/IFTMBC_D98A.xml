<?xml version="1.0" encoding="ISO-8859-1" standalone="no"?>
<Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.blujaysolutions.com/2018/IFTMBC file:///C:/Users/jss/Documents/seacon/germany_interfaces/develop/iftmbc_D96B/IFTMBC_D96B.xsd" xmlns="http://www.blujaysolutions.com/2018/IFTMBC">
    <Interchange>
        <UNB>
            <F0020>20180704327110</F0020>
            <S001>
                <F0001>UNOA</F0001>
                <F0002>1</F0002>
            </S001>
            <S002>
                <F0004>EGDHAM</F0004>
            </S002>
            <S003>
                <F0010>ICT</F0010>
            </S003>
            <S004>
                <F0017>180704</F0017>
                <F0019>1310</F0019>
            </S004>
        </UNB>
        <UNZ>
            <F0036>1</F0036>
            <F0020>20180704327110</F0020>
        </UNZ>
        <Group>
            <Message_IFTMBC>
                <UNH>
                    <F0062>560800426466</F0062>
                    <S009>
                        <F0065>IFTMBC</F0065>
                        <F0052>D</F0052>
                        <F0054>98A</F0054>
                        <F0051>UN</F0051>
                        <F0057>EGDHBG</F0057>
                    </S009>
                </UNH>
                <BGM>
                    <F1225>9</F1225>
                    <C002>
                        <F1001>770</F1001>
                    </C002>
                    <C106>
                        <F1004>560800426466</F1004>
                    </C106>
                </BGM>
                <DTM>
                    <C507>
                        <F2005>137</F2005>
                        <F2380>201807041310</F2380>
                        <F2379>203</F2379>
                    </C507>
                </DTM>
                <TSR>
                    <C536>
                        <F4065>30</F4065>
                    </C536>
                </TSR>
                <TCC>
                    <C200>
                        <F4237>P</F4237>
                    </C200>
                </TCC>
                <UNT>
                    <F0074>27</F0074>
                    <F0062>560800426466</F0062>
                </UNT>
                <SG1>
                    <LOC>
                        <F3227>73</F3227>
                        <C517>
                            <F3225>DEHAM</F3225>
                            <F3055>6</F3055>
                        </C517>
                    </LOC>
                </SG1>
                <SG1>
                    <LOC>
                        <F3227>91</F3227>
                        <C517>
                            <F3225>DEHAM</F3225>
                            <F3055>6</F3055>
                        </C517>
                    </LOC>
                </SG1>
                <SG2>
                    <RFF>
                        <C506>
                            <F1153>BN</F1153>
                            <F1154>560800426466</F1154>
                        </C506>
                    </RFF>
                    <DTM>
                        <C507>
                            <F2005>179</F2005>
                            <F2380>20180703</F2380>
                            <F2379>102</F2379>
                        </C507>
                    </DTM>
                </SG2>
                <SG2>
                    <RFF>
                        <C506>
                            <F1153>CT</F1153>
                            <F1154>SQK500212</F1154>
                        </C506>
                    </RFF>
                </SG2>
                <SG2>
                    <RFF>
                        <C506>
                            <F1153>RF</F1153>
                            <F1154>807/USFA/00253</F1154>
                        </C506>
                    </RFF>
                </SG2>
                <SG2>
                    <RFF>
                        <C506>
                            <F1153>FF</F1153>
                            <F1154>169124</F1154>
                        </C506>
                    </RFF>
                </SG2>
                <SG3>
                    <TDT>
                        <F8051>20</F8051>
                        <F8028>0986-021E</F8028>
                        <C222>
                            <F8213>TMAN</F8213>
                            <F1131>146</F1131>
                            <F8212>THALASSA MANA</F8212>
                        </C222>
                    </TDT>
                    <DTM>
                        <C507>
                            <F2005>133</F2005>
                            <F2380>20180712</F2380>
                            <F2379>102</F2379>
                        </C507>
                    </DTM>
                    <SG4>
                        <LOC>
                            <F3227>10</F3227>
                            <C517>
                                <F3225>DEHAM</F3225>
                                <F3055>6</F3055>
                            </C517>
                        </LOC>
                    </SG4>
                    <SG4>
                        <LOC>
                            <F3227>13</F3227>
                            <C517>
                                <F3225>LKCMB</F3225>
                                <F3055>6</F3055>
                            </C517>
                        </LOC>
                    </SG4>
                    <SG4>
                        <LOC>
                            <F3227>9</F3227>
                            <C517>
                                <F3225>DEHAM</F3225>
                                <F3055>6</F3055>
                            </C517>
                        </LOC>
                    </SG4>
                    <SG4>
                        <LOC>
                            <F3227>12</F3227>
                            <C517>
                                <F3225>INMUN</F3225>
                                <F3055>6</F3055>
                            </C517>
                        </LOC>
                    </SG4>
                    <SG4>
                        <LOC>
                            <F3227>7</F3227>
                            <C517>
                                <F3225>INMUN</F3225>
                                <F3055>6</F3055>
                            </C517>
                        </LOC>
                    </SG4>
                </SG3>
                <SG6>
                    <NAD>
                        <F3035>CZ</F3035>
                        <F3251>41468</F3251>
                        <F3207>DE</F3207>
                        <C082>
                            <F3039>DEI000109</F3039>
                        </C082>
                        <C058>
                            <F3124>ICT INTERNATIONALE CONTAINER TRANSP</F3124>
                            <F3124>ORT GMBH</F3124>
                            <F3124>SPERBERWEG 6 A</F3124>
                        </C058>
                    </NAD>
                </SG6>
                <SG9>
                    <GID>
                        <F1496>1</F1496>
                        <C213>
                            <F7224>1</F7224>
                            <F7065>22G0</F7065>
                            <F3055>87</F3055>
                        </C213>
                    </GID>
                    <TMD>
                        <F8341>2</F8341>
                        <C219>
                            <F8335>3</F8335>
                        </C219>
                    </TMD>
                    <LOC>
                        <F3227>98</F3227>
                        <C517>
                            <F3225>DEHBGH</F3225>
                            <F3055>6</F3055>
                        </C517>
                    </LOC>
                    <FTX>
                        <F4451>AAA</F4451>
                        <C108>
                            <F4440>3 COLLI, POLYPROPYLENE</F4440>
                        </C108>
                    </FTX>
                    <SG11>
                        <MEA>
                            <F6311>AAE</F6311>
                            <C502>
                                <F6313>G</F6313>
                            </C502>
                            <C174>
                                <F6411>KGM</F6411>
                                <F6314>3370</F6314>
                            </C174>
                        </MEA>
                    </SG11>
                    <SG11>
                        <MEA>
                            <F6311>AAE</F6311>
                            <C502>
                                <F6313>AAW</F6313>
                            </C502>
                            <C174>
                                <F6411>MTQ</F6411>
                                <F6314>0</F6314>
                            </C174>
                        </MEA>
                    </SG11>
                </SG9>
            </Message_IFTMBC>
        </Group>
    </Interchange>
</Envelope>